cypress/e2e/awx/resources/inventoryGroup.cy.ts

```diff
@@ -527,16 +518,17 @@ describe('Inventory Groups', () => {
       });
       cy.getByDataCy('become_enabled').click();
       cy.clickButton(/^Next$/);
-      cy.getByDataCy('execution-environment-select-form-group').within(() => {
-        cy.getBy('[aria-label="Options menu"]').click();
-      });
-      cy.get('[data-ouia-component-type="PF5/ModalContent"]').within(() => {
-        cy.filterTableBySingleSelect('name', executionEnvironment.name);
-        cy.get('[data-ouia-component-id="simple-table"] tbody').within(() => {
-          cy.get('[data-cy="checkbox-column-cell"] input').click();
-        });
-        cy.clickButton(/^Confirm/);
-      });
+      cy.singleSelectByDataCy('executionEnvironment', executionEnvironment.name);
+      // cy.getByDataCy('execution-environment-select-form-group').within(() => {
```

Comment:
Do we need to keep the commented code?

Reply:
Sorry missed that thought I removed it, will remove it right now